doc/source/reference/moving_from_1.xx_to_2.0.rst

```diff
@@ -0,0 +1,117 @@
+This is a guide for people moving from NetworkX 1.XX to NetworkX 2.0
```

Comment:
It is rare to use a full sentence as title. We can use a noun phrase like `Migration guide from 1.x to 2.0`.